apps/studio/components/ui/AIAssistantPanel/AIAssistant.tsx

```diff
@@ -458,29 +463,50 @@ export const AIAssistant = ({ className }: AIAssistantProps) => {
                       <Info size={16} className="mt-0.5" />
                     </div>
                     <div>
-                      <p>
-                        Sorry, I'm having trouble responding right now. If the error persists while
-                        retrying, you may try clearing the conversation's messages and try again.
-                      </p>
+                      {isContextExceededError ? (
+                        <p>
+                          This conversation has become too long for the Assistant to process. Please
+                          start a new chat to continue.
+                        </p>
+                      ) : (
+                        <p>
+                          Sorry, I'm having trouble responding right now. If the error persists
+                          while retrying, you may try clearing the conversation's messages and try
+                          again.
```

Comment:
come to think of it - we've removed the ability to clear messages in a conversation right?
there's only starting a new chat?

Reply:
you can delete messages instead of clearing but yes good point let's reframe this copy too